Source/UI/PaymentForm/ViewController/PaymentViewController.swift

```diff
@@ -143,7 +143,7 @@ final class PaymentViewController: UIViewController {
 
   @objc private func keyboardWillShow(notification: Notification) {
     guard let userInfo = notification.userInfo,
-      let keyboardFrameValue = userInfo[UIResponder.keyboardFrameBeginUserInfoKey] as? NSValue else { return }
+    let keyboardFrameValue = userInfo[UIResponder.keyboardFrameEndUserInfoKey] as? NSValue else { return }
```

Comment:
<img width="468" alt="Screenshot 2022-10-17 at 09 10 48" src="https://user-images.githubusercontent.com/59569588/196210095-6d5cbf50-864e-483b-8c7e-36886a686601.png">
Assume if fixes the above ?

Reply:
Yes, that's the one. I'm not sure why, but sometimes the start frame height is 100 pixels shorter than end height. I'm surprised the start was not 0 since you'd expect the animation to start from nothing 🤷‍♂️